src/snowcli/cli/project/schema.py

```diff
@@ -81,10 +81,22 @@ def as_fully_optional(self) -> RelaxedMap:
     }
 )
 
+StreamlitMapping = RelaxedMap(
+    {
+        "name": Str(),
+        "file": FilePath(),
+        Optional("stage"): Str(),
+        Optional("environment_file"): FilePath(),
+        Optional("pages_dir"): FilePath(),
```

Comment:
I think the stage should be neccessary. Using implicit default for creating objects on users account (as we do today) does not feel ok. 

I'm not sure about the environment file, it feels as required too.

cc @sfc-gh-zblackwood @sfc-gh-anugupta wdyt?

Reply:
Changed stage to required value